pyccel/stdlib/external/fitpack.py

```diff
@@ -1,5 +1,10 @@
-#$ header metavar print=True
+#------------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
+# go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
+#------------------------------------------------------------------------------------------#
+# pylint: disable=missing-module-docstring
```

Comment:
Please do not disable missing docstrings outside of the `tests/` folder

This file can have a docstring similar to that of dfftpack/lapack

@saidctb is this the correct link: http://www.netlib.org/dierckx/ ?

Reply:
I'm not sure anymore, probably lol